src/content/docs/query-your-data/explore-query-data/dashboards/prebuilt-dashboards.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Pre-built dasboards
+tags:
+  - Query your data
+  - Explore and query data
+  - Dashboards
+  - Pre-built dashboards
+---
+
+import dashboardsDashboardIndex from 'images/dashboards_screenshot-full_dashboard-index.png'
+
+import dashboardsPrebuiltViewAll from 'images/dashboards_screenshot-full_prebuilt-view-all.png'
+
+import dashboardsPrebuiltSearch from 'images/dashboards_screenshot-full_prebuilt-search.png'
+
+
+Through the New Relic UI, you can access a collection of pre-built dashboards based on high levels of user interest and activity. These pre-built dashboards enhance quickstart dashboard discovery, speed up the installation process, and get to instantly visualize your data.
+
+Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Dashboards** to open the dashboards page.
+
+<img
+  title="Dashboards index"
+  alt="Dashboards index"
+  src={dashboardsDashboardIndex}
+/>
+
+<figcaption>
+  Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Dashboards** to open the Dashboard index.
+</figcaption>
+
+
+## How to install pre-built dashboards [#use]
+
+To install a pre-built dashboard:
+
+1. From the dashboards page, search for the pre-built dashboard you want to customize.
+2. Click the pre-built dashboard for selecting it.
+3. Select the account for adding the dashboard and data you select.
+4. Follow the instructions you have in the **Setup data source** section. You can skip this step.
```

Comment:
Please remove "you have", so that the first sentence becomes:

`Follow the instructions in the **Setup data source** section.`

Reply:
What determines whether the user can skip this step?

Reply:
I'm not sure.. I added it because I saw it on the UI.